currency-v1/apiproxy/resources/jsc/ConfigureTargetRequest.js

```diff
@@ -1 +1,3 @@
 context.setVariable('target.copy.pathsuffix', false);
+
+context.setVariable('something', 'blah');
```

Comment:
WHAT IS THE POINT OF THIS CHANGE? ARE YOU CRAZY?

Reply:
hilarious. you're meant to fork this repo!